src/components/Editor/Editor.css

```diff
@@ -3,6 +3,8 @@
   --debug-expression-background: rgba(202, 227, 255, 0.5);
   --editor-searchbar-height: 27px;
   --editor-second-searchbar-height: 27px;
+  --debug-line-error-border: rgb(255, 0, 0);
+  --debug-expression-error-background: rgba(255, 127, 127, 0.5);
```

Comment:
Not sure which color to use as background.
This looked ok for me...

Reply:
- the background feels a bit strong , maybe lighter, or no background with just the red border lines

Reply:
Let's do something light :)